__tests__/Itinerary.tests.js

```diff
@@ -0,0 +1,22 @@
+const Itinerary = require("../src/Itinerary");
+const Port = require("../src/Port");
+
+describe('Itinerary', () => {
+  let liverpool;
+  let newYork;
+  let itinerary;
+
+  beforeEach(() => {
+    liverpool = new Port('Liverpool');
+    newYork = new Port('New York'); 
+    itinerary = jest.fn();
+  });
+
+  it('can be instantiated', () => {
+    expect(new Itinerary()).toBeInstanceOf(Object);
+  })
```

Comment:
Add a blank line for readability:
```suggestion
  })

```